### DISCUSSION TITLE
Categories not saving.......

### DISCUSSION THREAD

**Discussion by shansen008** [nextcloud/cookbook#2546] [FAQ and user questions]
Aloha all,

Is anyone else having a problem getting Categories to save?  I love the app so far and have imported lots of recipes from various websites.  I noticed that there were a few different categories that were automatically created during the import process.

How to create new categories wasnt very intuitive from the gui, but I eventually figured out that I could type them in while editing the recipes.  So I consolidated them into one "Main Dishes" category by deleting the others and saving each recipe.

So I decided that some of them I wanted to put under the "Smoker" category, so i typed that in and hit save, but it doesnt create the category at all.  The only way I can get it to create the category is if I delete all other categories in the recipe, then type out the one I want.  Its as though the app is only saving the first category in the list.

Is this a known bug or is there something wrong with my installation?

Im running nextcloud on unraid/docker.

Thanks!

**Comment by christianlupus** [comment]
Hello. 

Yes this is sort of expected. The current data model restricts the recipe to have one category maximum. (Honestly, the question of this is a good decision is something completely different.)

Nevertheless, the UI should only allow you to select a single category instead of multiple ones. I am currently abroad but if this is not automatically cleared, you could file a bug report to track this issue. 

Chris
